src/content/pages/faq.mdx

```diff
@@ -20,7 +20,7 @@ subtitle: Frequently Asked Questions about EuroPython
 </ul>
 
 <ul className="milestone-todo">
-  <li>**January/February 2025** Financial Aid Programme starts</li>
+  <li>**7 March 2025** Financial Aid Programme starts</li>
```

Comment:
```suggestion
  <li>**7 March 2025** [Financial Aid](/finaid/) Programme starts</li>
```

This should ideally move up to the milestone-done list, because by the time you merge, it would have been done :blush: Sorry for not being clear!

Also, down in line 29, since you already know the closing date, I suppose you could replace the TBD date with  `**20 April 2025:** Financial Aid Round One closes` (no need to link the /finaid again). But you could always update the milestone later of course!